Description: disable execution of munin-limits during cron jobs
Author: Lars Kruse <devel@sumpfralle.de>
Bug: https://github.com/munin-monitoring/munin/issues/1133
Last-Update: 2018-12-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/script/munin-cron.PL
+++ b/script/munin-cron.PL
@@ -21,5 +21,6 @@ $MUNIN_BINDIR/munin-update \$\@ || exit
 # Check limits defined for each service, and notify contacts.
 # This also writes state read by munin-httpd.
 
-$MUNIN_BINDIR/munin-limits \$\@
+# "munin-limits" is not yet working in 2.999.10 (to be done for the 3.0 release)
+#$MUNIN_BINDIR/munin-limits \$\@
 EOF
